test/resources/case9_templates/case9-test-policy_copy.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: case9-test-policy-copy
+spec:
+  remediationAction: inform
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: case9-test-configpolicy
+        spec:
+          remediationAction: inform
+          namespaceSelector:
+            exclude: ["kube-*"]
+            include: ["default"]
```

Comment:
Nit: using a namespace selector when the resource has a namespace is unnecessary. And in this case, the namespace in the resource does not satisfy the selector. I'm surprised that works, and I think it's probably ok that it works, but it is that little bit extra confusing.

Reply:
Yes, confusing is correct so I have removed them.